src/lidar_mini.hpp

```diff
@@ -18,28 +18,40 @@
  */
 class LIDARmini {
   private:
-    hwlib::pin_in &RX;
+    UARTConnection uart;
 
   public:
     /**
      * @brief Constructor
      */
-    LIDARmini(hwlib::pin_in &RX);
+    explicit LIDARmini();
```

Comment:
Explicit is only necessary if the constructor has exactly one parameter (because the compiler then thinks it is the copy constructor in some cases).